packages/grant-explorer/src/features/round/ViewCartPage/RoundInCart.tsx

```diff
@@ -15,14 +23,84 @@ export function RoundInCart(
     String(props.roundCart[0].chainId),
     props.roundCart[0].roundId
   ).round;
+
   const minDonationThresholdAmount =
     round?.roundMetadata?.quadraticFundingConfig?.minDonationThresholdAmount ??
     1;
+
+  const { address } = useAccount();
+  const votingTokenForChain = useCartStorage((state) =>
+    state.getVotingTokenForChain(props.roundCart[0].chainId)
+  );
+
+  const {
+    data: matchingEstimates,
+    error: matchingEstimateError,
+    isLoading: matchingEstimateLoading,
+  } = useMatchingEstimates([
+    {
+      roundId: getAddress(round?.id ?? zeroAddress),
+      chainid: props.roundCart[0].chainId,
+      potentialVotes: props.roundCart.map((proj) => ({
+        amount: parseUnits(
+          proj.amount ?? "0",
+          votingTokenForChain.decimal ?? 18
+        ),
+        recipient: proj.recipient,
+        contributor: address ?? zeroAddress,
+        token: votingTokenForChain.address.toLowerCase(),
+      })),
+    },
+  ]);
+
+  const estimateText = matchingEstimates
+    ?.flat()
+    .map((est) => est.differenceInUSD ?? 0)
+    .filter((diff) => diff > 0)
+    .reduce((acc, b) => acc + b, 0)
+    .toFixed(2);
+
+  const { passportState, passport } = usePassport({
+    address: address ?? "",
+  });
+
+  const isNotEligibleForMatching =
+    passportState === PassportState.NOT_CONNECTED ||
+    (passport?.score !== undefined && Number(passport.score) < 1);
+
   return (
     <div className="my-4 bg-grey-50 rounded-xl">
-      <div className="flex flex-row pt-4 sm:px-4 px-2">
-        <p className="text-xl  font-semibold">{round?.roundMetadata?.name}</p>
-        <p className="text-lg font-bold ml-2">({props.roundCart.length})</p>
+      <div className="flex flex-row items-center pt-4 sm:px-4 px-2 justify-between">
+        <div className={"flex"}>
```

Comment:
Lint rule proposal for the future: anything that makes us use either `attr="foo"` or `attr={"foo"}` all the time.